integrations/tensorflow/e2e/strings_test.py

```diff
@@ -33,12 +33,12 @@ def print_ids(self, ids):
     string_tensor = tf.strings.as_string(ids)
     tf.print(string_tensor)
 
-  @tf.function(input_signature=[
-      tf.TensorSpec((None, None), dtype=tf.int32),
-  ])
-  def strings_to_ids(self, ids):
-    wps = tf.gather(self.wordparts, ids)
-    return tf.strings.reduce_join(wps, 1)
+#  @tf.function(input_signature=[
```

Comment:
intentional?

Reply:
Unfortunately yes. It looks like gather/concat lowerings were either lost or not committed earlier. I may try to hack them in so we don't have the e2e tests for strings incomplete.